src/sentry/auth/helper.py

```diff
@@ -374,13 +375,13 @@ def _handle_unknown_identity(self, identity):
         if not op:
             if request.user.is_authenticated():
                 return self.respond('sentry/auth-confirm-link.html', {
-                    'identity': identity,
+                    'identity': InterfaceUser.to_python(identity),
```

Comment:
not totally sure this is a good idea, so let me know if there are better ways to do this